dts/bindings/spi/andestech.atcspi200.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2019, Andes Technology Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description:
+    This binding gives a representation of Andes ATCSPI200 SPI controller
+
+compatible: "andestech,atcspi200"
+
+include: spi-controller.yaml
+
+properties:
+
+    reg:
+      required: true
+
+    num-cs:
```

Comment:
Where is that being used?

Reply:
Thanks for reviewing.

Originally, we just hoped to let users know the number of CS our spi driver support. And due to our spi IP provide dedicated CS(automatic control by hardware), so we only support one dedicated CS pin.

After seeing your suggestion, we also think that it is not very suitable to be a property just for the purpose of providing information, so we decided to remove it, thank you very much.